packages/trace-viewer/src/ui/recorder/recorderView.tsx

```diff
@@ -280,7 +280,8 @@ const TraceView: React.FunctionComponent<{
     return snapshot.action || snapshot.after || snapshot.before;
   }, [action]);
   const snapshotUrls = React.useMemo(() => {
-    return snapshot ? extendSnapshot(snapshot) : undefined;
+    // TODO: Use actual setting. Requires settings UI to be wired up
```

Comment:
Does this mean the patch is not complete?

Reply:
It seemed to me that this Recorder trace view was not complete. It needs its own settings UI to integrate the canvas setting, and the menu doesn't trivially integrate with Recorder trace view because of annoying CSS issues.